### PR TITLE
feat(time-utilities): add TimerManager

### PR DESCRIPTION
**Pull request by Favna**

### CODE REVIEW

File: packages/time-utilities/src/lib/TimerManager.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+export class TimerManager extends null {
+	/**
+	 * A collection of timeouts to clear on destroy
+	 */
+	private static storedTimeouts: Set<NodeJS.Timeout> = new Set();
+
+	/**
+	 * A collection of intervals to clear on destroy
+	 */
+	private static storedIntervals: Set<NodeJS.Timeout> = new Set();
```
Comment (vladfrangu) [inline packages/time-utilities/src/lib/TimerManager.ts:13]:
```suggestion
	private static storedTimeouts = new Set<NodeJS.Timeout>();

	/**
	 * A collection of intervals to clear on destroy
	 */
	private static storedIntervals = new Set<NodeJS.Timeout>();
```

File: packages/time-utilities/src/lib/TimerManager.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+	/**
+	 * Set a timeout that Project Blue can clear when destroyed
```
Comment (vladfrangu) [inline packages/time-utilities/src/lib/TimerManager.ts:16]: pardon my blue what

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: packages/time-utilities/src/lib/TimerManager.ts (hunk 2 above)
Comment (Favna) [reply packages/time-utilities/src/lib/TimerManager.ts:16]: lol oops

Comment (Favna) [reply packages/time-utilities/src/lib/TimerManager.ts]: resolved in 744e589

File: packages/time-utilities/src/lib/TimerManager.ts (hunk 1 above)
Comment (Favna) [reply packages/time-utilities/src/lib/TimerManager.ts]: resolved in 744e589

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**